src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/SpecialFileProviders/AbstractSpecialFileProvider.cs

```diff
@@ -107,7 +105,11 @@ public async Task<string> GetFileAsync(SpecialFiles fileId, SpecialFileFlags fla
             }
 
             // We haven't found the file but return the default file path as that's the contract.
-            IProjectTree rootNode = _projectTree.CurrentTree;
+            IProjectTree? rootNode = _projectTree.CurrentTree;
+
+            if (rootNode == null)
+                return null;
```

Comment:
This is a behaviour change. Instead of NRE it returns null. There were other ways this could return null however, so I don't think it's a problem.